src/utils/swapQuotes.ts

```diff
@@ -26,15 +43,11 @@ export async function swapQuotes(
   }
 
   // Create an EdgeSwapRequest
-  // NOTE: This is a placeholder that will be replaced in the future
-  const swapRequest = {
-    fromWallet: currencyWallets[0],
-    toWallet: currencyWallets[1],
-    fromCurrencyCode: currencyWallets[0].currencyInfo.currencyCode,
-    toCurrencyCode: currencyWallets[1].currencyInfo.currencyCode,
-    nativeAmount: '1000000',
-    quoteFor: 'from' as const
-  }
+  const swapRequest = createFromSwapRequest(
+    currencyWallets[0],
+    currencyWallets[1],
+    1000000
```

Comment:
I would recommend using a constant for this number (maybe for the 0 and 1 as well).  As it is, it is a little unclear what the number means or represents.